backend/graphql/user.graphql

```diff
@@ -14,4 +14,6 @@ type User {
     profile_metadata: ProfileMetadata
     submissions: [Submission]! @belongsToMany
     pivot: SubmissionUser
+    inline_comments: [InlineComment]! @hasMany(relation: "inlineCOmments")
+    overall_comments: [OverallComment]! @hasMany(relation: "overallComments")
```

Comment:
```suggestion
```
This wasn't part of the spec.  Is there a specific UI case you're imagining here?

IF there were a valid case for this, this would need to be a paginated query as it will potentially be a large amount of records returned.

Reply:
I misunderstood. I figured because we were establishing a direct relationship between inline and overall comments to users that it made sense to establish the inverse of this relationship as well. I'll accept your suggestions.